### PR TITLE
Update UG for delete function

### LINKED ISSUE #29: Write guide for remove function for User Guide

### PR DESCRIPTION
**Pull request by Lusi711**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -55,11 +57,29 @@ Added: coding
...
+```
+Noted. I've removed this task:
+[x] coding (by: Oct 19 2020 07.00 PM)
+	Now you have 0 tasks in the list.
+```
 
 ## Notebook Mode
-In `Notebook mode`, you can `add`/`select` a `notebook`, `section` or `page`.
+In `Notebook mode`, you can manage a shelf of notebooks. 
+Each notebook contains sections, and each section contains pages. Each page holds your type-written notes.
```
Comment (franceneee) [inline docs/UserGuide.md:79]: oh yes hehehe that's a better description than mine xD

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-    - [Add a `section`](https://ay2021s1-cs2113t-t12-3.github.io/tp/UserGuide.html/#adding-a-section)
-    - [Add a `page`](https://ay2021s1-cs2113t-t12-3.github.io/tp/UserGuide.html/#adding-a-page)
-    - [`Select`](https://ay2021s1-cs2113t-t12-3.github.io/tp/UserGuide.html/#select)
-- [Command Summary](https://ay2021s1-cs2113t-t12-3.github.io/tp/UserGuide.html/#command-summary)
+- [Quick Start](#quick-start)
+- [Features](#features)
+  - [Mode Switch: `mode`](#mode-switch)
+  - [Planner Mode](#planner-mode)
+    - [Adding task: `add`](#adding-a-task)
+    - [Deleting task: `delete`]
```
Comment (franceneee) [inline docs/UserGuide.md:9]: missing the link to the delete function!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -134,8 +193,14 @@ In a selected *SECTION*,
...
+
+### Notebook Mode
 **Command** | **Format** | **Example**
 ----------- | ---------- | -----------
-*TIMETABLE MODE* `add` | add /tTASK /by[dd/MM/yyyy] [hhmm] | add /tcoding /by19-10-2020 1705
-*NOTEBOOK MODE* `add` | 1) add /nNOTEBOOK 2) add /sSECTION 3) add /pPAGE; CONTENT | add /nCS2101
-*NOTEBOOK MODE* `select` | 1) select /nNOTEBOOK 2) select /sSECTION 3) select /pNUMBER | select /nCS2101
+Add: `add` | 1) add /nNOTEBOOK 2) add /sSECTION 3) add /pPAGE; CONTENT | add /nCS2101
+Select: `select` | 1) select /nNOTEBOOK 2) select /sSECTION 3) select /pNUMBER | select /nCS2101
+Delete: `delete` | 1) select /nNOTEBOOK /sSECTION /pNUMBER | select /nCS2113T /sW10 /p1
```
Comment (franceneee) [inline docs/UserGuide.md:206]: oh yes separating the commands for the different modes seems like a good idea! 😀

**Review by franceneee — APPROVED** [review]
great job i think it's good to merge! maybe just add in the link to the section in the navigation panel?

**Comment by longngng** [comment]
I feel like we should highlight the commands only. 
image
For example, in this pic, I think can change Timetable, task, deadline to another mark-up style, like italic. 
Other than that, it looks good.
Sorry this was from Francene's commit haha.

File: docs/UserGuide.md (hunk 3 above)
Comment (longngng) [reply docs/UserGuide.md:206]: Totally agree!

**Pull request merged**